### PR TITLE
Os 18 custom help block module

### PR DESCRIPTION
**Pull request by SanyokMedved**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

## Steps for review

- [ ] Please provide steps for review here.

### CODE REVIEW

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
ok to test

File: modules/custom/advanced_help_block/advanced_help_block.info.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Advanced Help Block
+type: module
+description: Module for advanced help blocks which can be shown on the selected pages.
+core: 8.x
+package: Custom
```
Comment (podarok) [inline modules/custom/advanced_help_block/advanced_help_block.info.yml]: please introduce version

File: modules/custom/advanced_help_block/config/install/field.field.advanced_help_block.advanced_help_block.field_ahb_video.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+third_party_settings:
+  datalayer:
+    expose: 0
+    label: field_ahb_video
+id: advanced_help_block.advanced_help_block.field_ahb_video
+field_name: field_ahb_video
+entity_type: advanced_help_block
+bundle: advanced_help_block
+label: Video
+description: ''
```
Comment (podarok) [inline modules/custom/advanced_help_block/config/install/field.field.advanced_help_block.advanced_help_block.field_ahb_video.yml]: Could you fill in all descriptions for the end users?

File: modules/custom/advanced_help_block/library/js/ahb.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+(function ($, Drupal, drupalSettings) {
+    'use strict';
+
+    Drupal.behaviors.youtubeModal = {
+        attach: function (context, settings) {
+            $(".youtube-link").grtyoutube(
+                {
+                    autoPlay: true,
+                    theme: "dark" // or dark
```
Comment (podarok) [inline modules/custom/advanced_help_block/library/js/ahb.js]: comment makes no sense

File: modules/custom/advanced_help_block/library/js/grt-youtube-popup.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+ * Version: 1.0
+ * Author: GRT107
+ *
+ * Copyright (c) 2017 GRT107
+ * Released under the MIT license
+*/
+
+(function ( $ ) {
+
+    $.fn.grtyoutube = function( options ) {
```
Comment (podarok) [inline modules/custom/advanced_help_block/library/js/grt-youtube-popup.js]: Can we add it via composer?
It has MIT LIcense which needs change of OpenY license when code is within repository

File: modules/custom/advanced_help_block/templates/advanced-help-block-list-render.html.twig [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{{ attach_library('advanced_help_block/ahb_modal') }}
```
Comment (podarok) [inline modules/custom/advanced_help_block/templates/advanced-help-block-list-render.html.twig:16]: Docblock, please

File: modules/openy_features/openy_demo_content/modules/openy_demo_ahb/openy_demo_ahb.info.yml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+name: 'OpenY Demo Advanced Help Blocks'
+description: 'OpenY Demo Advanced Help Blocks.'
+type: module
+core: 8.x
+package: OpenY
```
Comment (podarok) [inline modules/openy_features/openy_demo_content/modules/openy_demo_ahb/openy_demo_ahb.info.yml:5]: add version, please

**Review by podarok — CHANGES_REQUESTED** [review]
We have to fix license issue

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build315 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build315 |
| Installation process | http://install.openy-dev.ffwua.com/build315/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/315/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build315/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build315/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build315/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build315/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build315/blogWCAG2AAhtmlcs.txt

File: modules/custom/advanced_help_block/advanced_help_block.info.yml (hunk 1 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/advanced_help_block.info.yml]: done.

File: modules/custom/advanced_help_block/config/install/field.field.advanced_help_block.advanced_help_block.field_ahb_video.yml (hunk 2 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/config/install/field.field.advanced_help_block.advanced_help_block.field_ahb_video.yml]: done.

File: modules/custom/advanced_help_block/library/js/ahb.js (hunk 3 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/library/js/ahb.js]: done.

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build316 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build316 |
| Installation process | http://install.openy-dev.ffwua.com/build316/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/316/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build316/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build316/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build316/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build316/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build316/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build317 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build317 |
| Installation process | http://install.openy-dev.ffwua.com/build317/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/317/

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build318 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build318 |
| Installation process | http://install.openy-dev.ffwua.com/build318/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/318/

File: modules/openy_features/openy_demo_content/modules/openy_demo_ahb/openy_demo_ahb.info.yml (hunk 6 above)
Comment (SanyokMedved) [reply modules/openy_features/openy_demo_content/modules/openy_demo_ahb/openy_demo_ahb.info.yml:5]: done.

File: modules/custom/advanced_help_block/templates/advanced-help-block-list-render.html.twig (hunk 5 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/templates/advanced-help-block-list-render.html.twig:16]: done.

**Comment by kolesnikoff** [comment]
@podarok please also review https://github.com/ymcatwincities/openy-project/pull/31 as a part of this task

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build319 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build319 |
| Installation process | http://install.openy-dev.ffwua.com/build319/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/319/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build319/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build319/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build319/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build319/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build319/blogWCAG2AAhtmlcs.txt

**Comment by podarok** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build340 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build340 |
| Installation process | http://install.openy-dev.ffwua.com/build340/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/340/

**Comment by podarok** [comment]
image
@kolesnikoff please provide steps for review and check why URI is not working ( I assume because of missed leading slash )

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build344 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build344 |
| Installation process | http://install.openy-dev.ffwua.com/build344/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/344/

**Comment by SanyokMedved** [comment]
@podarok please check last commits. I've added module and migration into profile. Thank you

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build344/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build344/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build344/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build344/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build344/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build345/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build345/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build345/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build345/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build345/blogWCAG2AAhtmlcs.txt

**Comment by podarok** [comment]
retest this please

File: modules/custom/advanced_help_block/advanced_help_block.install [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+ * @file
+ * Contains install and update routines.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function advanced_help_block_install() {
+  \Drupal::configFactory()->getEditable('advanced_help_block.settings')
+    ->set('advanced_help_block.view_type', \Drupal::config('advanced_help_block.settings')->get('advanced_help_block.view_type'))
```
Comment (podarok) [inline modules/custom/advanced_help_block/advanced_help_block.install]: HIghly confusing code
Why needed to set same data from itself?
Looks like some sort of miscoding here

File: modules/custom/advanced_help_block/advanced_help_block.module [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+ * Implements hook_help().
+ */
+function advanced_help_block_help($path, $arg) {
+  $query = \Drupal::entityQuery('advanced_help_block');
+  $entity_ids = $query->execute();
+  $ids = [];
+  foreach ($entity_ids as $entity_id) {
+    $ids[] = $entity_id;
+  }
+  $blocks = AdvancedHelpBlock::loadMultiple($ids);
```
Comment (podarok) [inline modules/custom/advanced_help_block/advanced_help_block.module]: What if 1000 ids are in a system?
We'll brake the site with memory limit, though

Let's split by chunks by 50 maximum and load 50 only per loop

File: modules/custom/advanced_help_block/advanced_help_block.module [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+          Url::fromUri($block->get('field_ahb_video')->value, [
+            'attributes' => [
+              'youtubeid' => $url_params['v'],
+              'class' => ['youtube-link'],
+            ],
+          ])
+        )->toString();
+      }
+      $output[$block->id()] = [
+        'title' => $block->get('field_ahb_title')->value,
```
Comment (podarok) [inline modules/custom/advanced_help_block/advanced_help_block.module]: Use magic everywhere if you started to use it somewhere
```$block->field_ahb_title->value```

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build356 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build356 |
| Installation process | http://install.openy-dev.ffwua.com/build356/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/356/

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockForm.php [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+ *
+ * @ingroup advanced_help_block
+ */
+class AdvancedHelpBlockForm extends ContentEntityForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    /* @var $entity \Drupal\advanced_help_block\Entity\AdvancedHelpBlock */
```
Comment (podarok) [inline modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockForm.php]: comment is not from this namespace, please remove or fix

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build356/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build356/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build356/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build356/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build356/blogWCAG2AAhtmlcs.txt

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+  /**
+   * Form submission handler.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param FormStateInterface $form_state
+   *   An associative array containing the current state of the form.
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    $config_factory = \Drupal::configFactory();
```
Comment (podarok) [inline modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php]: Let's get used to Drupal 8 ecosystem
Here is an example of container injection into form https://github.com/ymcatwincities/openy/blob/8.x-1.x/modules/custom/openy_calc/src/Form/CalcBlockForm.php#L48
Please avoid usage ```\Drupal::anything``` in OOP code

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+   * @return array
+   *   Form definition array.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param FormStateInterface $form_state
+   *   An associative array containing the current state of the form.
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config_factory = \Drupal::configFactory();
```
Comment (podarok) [inline modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php]: Let's get used to Drupal 8 ecosystem

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php (hunk 11 above)
Comment (podarok) [reply modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php]: Also, here is an example of ConfigFormBase, which was created specifically for config manipulations
https://www.drupal.org/docs/8/api/form-api/configformbase-with-simple-configuration-api
https://www.drupal.org/docs/8/api/configuration-api/working-with-configuration-forms

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+
+/**
+ * Class AdvancedHelpBlockSettingsForm.
+ *
+ * @package Drupal\advanced_help_block\Form
+ * @ingroup advanced_help_block
+ */
+class AdvancedHelpBlockSettingsForm extends FormBase {
```
Comment (podarok) [inline modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php]: No need to create a dedicated form, use block form from core, please http://cgit.drupalcode.org/examples/tree/block_example/src/Plugin/Block/ExampleConfigurableTextBlock.php

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+      '#open' => TRUE,
+    ];
+    $form['advanced_help_block']['view_type'] = [
+      '#type' => 'select',
+      '#title' => t('Output advanced help block as:'),
+      '#options' => [
+        'block' => t('Block'),
+        'message' => t('Message')
+      ],
+      '#default_value' => $config_factory->getEditable('advanced_help_block.settings')
```
Comment (podarok) [inline modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php]: use drupal 8 ecosystem, please

**Review by podarok — CHANGES_REQUESTED** [review]
Let's use Drupal 8 for full power. Othersiwe our code won't be viable soon with 8.6+

File: modules/custom/advanced_help_block/advanced_help_block.module (hunk 8 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/advanced_help_block.module]: File is removed.

File: modules/custom/advanced_help_block/advanced_help_block.module (hunk 9 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/advanced_help_block.module]: I changed all code on this your recommendation. Thank you.

File: modules/custom/advanced_help_block/library/js/grt-youtube-popup.js (hunk 4 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/library/js/grt-youtube-popup.js]: Added with composer.

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockForm.php (hunk 10 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockForm.php]: fixed

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php (hunk 13 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php]: This is not configuration for block, as you recommended. This is configuration form for module. I think this recommendation is misunderstanding. Not fixed, Please check again.

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php (hunk 12 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php]: changed

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php (hunk 14 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php]: changed

File: modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php (hunk 11 above)
Comment (SanyokMedved) [reply modules/custom/advanced_help_block/src/Form/AdvancedHelpBlockSettingsForm.php]: Changed. I've left only \Drupal::currentUser()->id() in AdvancedHelpBlock.php in preCreate function. 

Config form was changed.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build366 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build366 |
| Installation process | http://install.openy-dev.ffwua.com/build366/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/366/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build366/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build366/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build366/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build366/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build366/blogWCAG2AAhtmlcs.txt

**Comment by podarok** [comment]
retest this please

**Review by podarok — CHANGES_REQUESTED** [review]
Could you please fix conflict?

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build393 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build393 |
| Installation process | http://install.openy-dev.ffwua.com/build393/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/393/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build393/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build393/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build393/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build393/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build393/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build394 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build394 |
| Installation process | http://install.openy-dev.ffwua.com/build394/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/394/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build394/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build394/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build394/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build394/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build394/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build395 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build395 |
| Installation process | http://install.openy-dev.ffwua.com/build395/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/395/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build395/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build395/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build395/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build395/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build395/blogWCAG2AAhtmlcs.txt

**Pull request merged**